libs/journeys/ui/src/components/StepFooter/FooterButtonList/FooterButtonList.spec.tsx

```diff
@@ -16,4 +16,14 @@ describe('FooterButtonList', () => {
     )
     expect(getAllByTestId('ShareIcon')).toHaveLength(1)
   })
+
+  it('should show like and dislike button', () => {
+    const { getAllByTestId } = render(
+      <SnackbarProvider>
+        <FooterButtonList />
+      </SnackbarProvider>
+    )
+    expect(getAllByTestId('ThumbsUpIcon')).toHaveLength(1)
+    expect(getAllByTestId('ThumbsDownIcon')).toHaveLength(1)
```

Comment:
My bad, set the wrong pattern here due to how it was originally being rendered

```suggestion
    expect(getAllByTestId('ThumbsUpIcon')).toBeInTheDocument()
    expect(getAllByTestId('ThumbsDownIcon')).toBeInTheDocument()
```